core/templates/domain/learner_dashboard/learner-playlist-modal.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="modal-header">
+  <h3 translate="I18N_LEARNER_DASHBOARD_REMOVE_ACTIVITY_MODAL_HEADER" translate-values="{sectionNameI18nId: sectionNameI18nId}"></h3>
+</div>
+<div class="modal-body">
+  <p translate="I18N_LEARNER_DASHBOARD_REMOVE_ACTIVITY_MODAL_BODY" translate-values="{entityTitle: activityTitle, sectionNameI18nId: sectionNameI18nId}"></p>
```

Comment:
Did you test this locally? The translate tags need to be migrated to use pipes in Angular.

Reply:
Sorry, it was a misunderstanding.
Yes!